pkg/codegen/testing/test/testdata/regress-py-12546/python/pulumi_plant/tree/v1/rubber_tree.py

```diff
@@ -58,6 +58,9 @@ def __init__(__self__, *,
     @property
     @pulumi.getter
     def diameter(self) -> pulumi.Input['Diameter']:
+        warnings.warn("""Dear future maintainer, if there are changes here, make sure that this is printed before the value is set to the default or else this will always print.""", DeprecationWarning)
```

Comment:
Set to the default? Is this because it uses the same deprecation message as the input?

Reply:
Correct. This was added to track a regression and dutifully spooked me on first read too (even though I added it).